pages/home.tsx

```diff
@@ -17,10 +80,121 @@ const Home = ({ user }: ServerPageProps) => {
-          <Text>{user.role}</Text>
+          {user.role === 'patient' ? (
+            <Tabs justify="start">
+              <Tab title="Ready for Pickup">
+                <Box pad="medium">
+                  {prescriptions.map(
+                    ({
+                      id,
+                      name,
+                      Location,
+                      LockerBox,
+                      balance,
+                      pickupCode
+                    }) => (
+                      <Card
+                        key={id}
+                        pad="medium"
+                        margin="medium"
+                        gap="medium"
+                        background={theme.global.colors['light-1']}
+                      >
+                        <CardHeader>{name}</CardHeader>
+                        <CardBody>
+                          <Box>
+                            <div>{Location.streetAddress}</div>
+                            <div>{Location.phoneNumber}</div>
+                            <div>Lockerbox: {LockerBox.label}</div>
+                            <Box align="start" pad="none">
+                              <Button
+                                label="QR Code"
+                                onClick={() => {
+                                  setQrCode(pickupCode)
+                                  setShowQrCode(true)
+                                }}
+                                style={{
+                                  background: theme.global.colors['status-ok'],
+                                  borderRadius: '4px'
+                                }}
+                                size="medium"
+                              />
+                            </Box>
+                          </Box>
+                        </CardBody>
+                        <CardFooter>
+                          <p>Amount Due: ${balance}</p>
+                        </CardFooter>
+                      </Card>
+                    )
+                  )}
+                </Box>
+              </Tab>
+              <Tab title="Previous Pickups"></Tab>
+            </Tabs>
+          ) : (
+            <>
+              {pharmacyLocation && (
+                <Card
+                  pad="medium"
+                  margin="medium"
+                  gap="medium"
+                  background={theme.global.colors['light-1']}
+                >
+                  <CardHeader>{pharmacyLocation.id}</CardHeader>
+                  <CardBody>
+                    <Box>
+                      <div>{pharmacyLocation.streetAddress}</div>
+                      <div>{pharmacyLocation.phoneNumber}</div>
+                      {/* <div>Lockerbox: {LockerBox.label}</div> */}
+                    </Box>
+                  </CardBody>
+                  <CardFooter>{/* <p>Amount Due: ${balance}</p> */}</CardFooter>
+                </Card>
+              )}
+            </>
+          )}
```

Comment:
This whole block is a pretty large conditional render based on the user role. I would suggest splitting it up into separate components. Refer to the settings page as an example 
https://github.com/pharmaboxfydp/pharmabox/blob/9ad02464ce50ab98909831a6b5d64b2fcfdb400a/pages/settings/%5B%5B...index%5D%5D.tsx#L52